pandas/tests/scalar/timestamp/test_assignment_dtype.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+from pandas.core.dtypes.dtypes import DatetimeTZDtype
+
+
+# referencing #19843: scalar assignment of a tz-aware is object dtype
```

Comment:
I think we can find a home for this in another test file (and don't need a new one).  In addition, the comment should be under the class declaration.

@jreback : I think `tests/frame/test_indexing.py` makes sense here.  What do you think?

Reply:
Sounds good, should they go under ```TestDataFrameIndexingDatetimeWithTZ```, or in their own class?

Reply:
Same class sounds good for now.